Classes/Search/SearchViewController.swift

```diff
@@ -89,6 +103,24 @@ SearchResultSectionControllerDelegate {
             collectionView.collectionViewLayout.invalidateLayout()
         }
     }
+    // MARK: Notifications
+
+    @objc
+    func onKeyboardWillShow(notification: NSNotification) {
+        guard let frame = notification.userInfo?[UIKeyboardFrameEndUserInfoKey] as? CGRect else { return }
+        let converted = view.convert(frame, from: nil)
+        let intersection = converted.intersection(frame)
+        let bottomInset = intersection.height - bottomLayoutGuide.length
+        let inset = UIEdgeInsets(top: 0, left: 0, bottom: bottomInset, right: 0)
+        collectionView.contentInset = inset
+        collectionView.scrollIndicatorInsets = inset
+    }
+
+    @objc
+    func onKeyboardWillHide(notification: NSNotification) {
+        collectionView.contentInset = .zero
+        collectionView.scrollIndicatorInsets = .zero
+    }
```

Comment:
Note to self: let's create a `UIScrollView` extension that does this automatically